spec/domain/formations_spec.exs

```diff
@@ -119,6 +133,19 @@ defmodule Squadster.Domain.FormationsSpec do
     end
   end
 
+  describe "create_squad_member/2" do
+    let :squad, do: insert(:squad)
+
+    before do
+      mock NormalizeQueue, :start_link
+    end
+
+    it "returns new squad_member" do
```

Comment:
Why you did not use `entities_count`?
And also, I think there has to be spec for users, who already have squad.

Reply:
Added a spec for users with squad in service tests